File: src/main/scala-graphql/io/github/graphql/model/MergeBranchMutationResponse.scala
package io.github.graphql.model

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult
import java.util.{ Map => JMap }

@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-06T19:48:01+0800"
)
class MergeBranchMutationResponse extends GraphQLResult[JMap[String, MergeBranchPayloadTO]] {

    def mergeBranch(): MergeBranchPayloadTO = {
        val data: JMap[String, MergeBranchPayloadTO] = getData
        if (data != null) data.get(MergeBranchMutationResponse.OPERATION_NAME) else null
    }

}

object MergeBranchMutationResponse {

    private final val OPERATION_NAME: String = "mergeBranch"

}

File: src/main/java-graphql/io/github/graphql/model/MergeBranchMutationResponse.java
package io.github.graphql.model;

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult;
import java.util.Map;

@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-06T19:47:48+0800"
)
public class MergeBranchMutationResponse extends GraphQLResult<Map<String, MergeBranchPayloadTO>> {

    private static final String OPERATION_NAME = "mergeBranch";

    public MergeBranchMutationResponse() {
    }

    public MergeBranchPayloadTO mergeBranch() {
        Map<String, MergeBranchPayloadTO> data = getData();
        return data != null ? data.get(OPERATION_NAME) : null;
    }

}
